code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+
```

Comment:
Currently there aren't any tests passing, once you have a chance to come back to them I will regrade for you! Remember to check out lab TA or teachers for help if needed!